Common/Utils/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-# Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+Common/Utils/CMakeLists.txt# Copyright 2019-2020 CERN and copyright holders of ALICE O2.
```

Comment:
Hi @hjbossi : there is a stray line here

Reply:
Okay, this should be resolved in the most recent commit!